repose-aggregator/docs/src/asciibinder/filters/openapi-validator.adoc

```diff
@@ -80,6 +83,29 @@ a| * An error occurs while attempting to read the request.
    * An unknown error occurs while validating schema from the OpenAPI document against the request.
 |===
 
+== Role Based Access Control
+For convenience an extension is provided that does role checking when used in combination with the <<keystone-v2.adoc#, Keystone v2 filter>>.
+This is enable through the extension mechanism of *Swagger/OpenApi* under the element name `x-rax-roles`.
+
+[source, yaml]
+.partial api-spec.yml
+----
+paths:
+  /pets:
+    get:
+      summary: List all pets
+      operationId: listPets
+      x-rax-roles: ["user", "admin"]  <1>
+----
+<1> The array of role names that are acceptable. This value must be an array even if it contains a single value.
```

Comment:
Two sentences on a single line.

`array` -> `sequence`? I think that's the right parlance.